crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -373,7 +399,7 @@ It can greatly improve performance (and readability) in such situations as it pr
                 time_ctrl_write.pause();
             }
 
-            for line in &time_series.lines {
+            for line in all_lines {
```

Comment:
for whatever reason this was called lines before, it's super confusing now that I review stuff here